modules/gapi/test/opencl/symm7x7_test.cl

```diff
@@ -0,0 +1,302 @@
+    a1_sum[1] = a1[1] + a1[5];
+    a1_sum[2] = a1[2] + a1[4];
+
+    a2_sum[0] = a2[0] + a2[6];
+    a2_sum[1] = a2[1] + a2[5];
+    a2_sum[2] = a2[2] + a2[4];
+
+    c0_sum[0] = c0[0] + c0[6];
+    c0_sum[1] = c0[1] + c0[5];
+    c0_sum[2] = c0[2] + c0[4];
+
+    c1_sum[0] = c1[0] + c1[6];
+    c1_sum[1] = c1[1] + c1[5];
+    c1_sum[2] = c1[2] + c1[4];
+
+    c2_sum[0] = c2[0] + c2[6];
+    c2_sum[1] = c2[1] + c2[5];
+    c2_sum[2] = c2[2] + c2[4];
+
+    b_sum[0] = b[0] + b[6];
+    b_sum[1] = b[1] + b[5];
+    b_sum[2] = b[2] + b[4];
+
+    float8 A = b[3];
+    float8 intermediate = A * (float)coeff[0];
+
+    float8 B = b_sum[2] +
+               (float8)(a2[3], b[3].s0123, b[3].s456) +
+               (float8)(b[3].s123, b[3].s4567, c0[3]);
+    intermediate += B * (float)coeff[1];
+
+    float8 C = (float8)(a2_sum[2], b_sum[2].s0123, b_sum[2].s456) +
+               (float8)(b_sum[2].s123, b_sum[2].s4567, c0_sum[2]);
+    intermediate += C * (float)coeff[2];
+
+    float8 D = b_sum[1] +
+               (float8)(a1[3], a2[3], b[3].s0123, b[3].s45) +
+               (float8)(b[3].s23, b[3].s4567, c0[3], c1[3]);
+    intermediate += D * (float)coeff[3];
+
+    float8 E = (float8)(a2_sum[1], b_sum[1].s0123, b_sum[1].s456) +
+              (float8)( b_sum[1].s123, b_sum[1].s4567, c0_sum[1]) +
+              (float8)( a1_sum[2], a2_sum[2], b_sum[2].s0123, b_sum[2].s45) +
+              (float8)( b_sum[2].s23, b_sum[2].s4567, c0_sum[2], c1_sum[2]);
+    intermediate += E * (float)coeff[4];
+
+    float8 F = (float8)(a1_sum[1], a2_sum[1], b_sum[1].s0123, b_sum[1].s45) +
+               (float8)(b_sum[1].s23, b_sum[1].s4567, c0_sum[1], c1_sum[1]);
+    intermediate += F * (float)coeff[5];
+
+    float8 G = b_sum[0] +
+               (float8)(a0[3], a1[3], a2[3], b[3].s0123, b[3].s4) +
+               (float8)(b[3].s3, b[3].s4567, c0[3], c1[3], c2[3]);
+    intermediate += G * (float)coeff[6];
+
+    float8 H = (float8)(a2_sum[0], b_sum[0].s0123, b_sum[0].s456) +
+               (float8)(b_sum[0].s123, b_sum[0].s4567, c0_sum[0]) +
+               (float8)(a0_sum[2], a1_sum[2], a2_sum[2], b_sum[2].s0123, b_sum[2].s4) +
+               (float8)(b_sum[2].s3, b_sum[2].s4567, c0_sum[2], c1_sum[2], c2_sum[2]);
+    intermediate += H * (float)coeff[7];
+
+    float8 I = (float8)(a1_sum[0], a2_sum[0], b_sum[0].s0123, b_sum[0].s45) +
+               (float8)(b_sum[0].s23, b_sum[0].s4567, c0_sum[0], c1_sum[0]) +
+               (float8)(a0_sum[1], a1_sum[1], a2_sum[1], b_sum[1].s0123, b_sum[1].s4) +
+               (float8)(b_sum[1].s3, b_sum[1].s4567, c0_sum[1], c1_sum[1], c2_sum[1]);
+    intermediate += I * (float)coeff[8];
+
+
+    float8 J = (float8)(a0_sum[0], a1_sum[0], a2_sum[0], b_sum[0].s0123, b_sum[0].s4) +
+               (float8)(b_sum[0].s3, b_sum[0].s4567, c0_sum[0], c1_sum[0], c2_sum[0]);
+    intermediate += J * (float)coeff[9];
+
+    intermediate *= SCALE;
+
+    vstore8(convert_uchar8_sat(intermediate), 0, (__global uchar*)(pdst));
+
+}
```

Comment:
I assumes it expects a newline at the end of this file

Reply:
Done

Reply:
Done